packages/host/app/components/operator-mode/publish-realm-modal.gts

```diff
@@ -468,6 +481,13 @@ export default class PublishRealmModal extends Component<Signature> {
     window.open(this.generatedUrl, '_blank');
   }
 
+  @action
+  handleOpenCustomSubdomainSite() {
+    if (this.claimedDomainPublishedUrl) {
+      window.open(this.claimedDomainPublishedUrl + 'index', '_blank');
+    }
+  }
```

Comment:
I failed to notice this before but I’d prefer these buttons be true links vs using `window.open`. That lets you hover over the button and see the URL. I don’t see any value in doing this via redirect. (Same for the above.)